src/app/GitUI/CommandsDialogs/BrowseDialog/DashboardControl/Dashboard.Designer.cs

```diff
@@ -46,7 +48,7 @@ private void InitializeComponent()
             // pnlLeft
             // 
             pnlLeft.Anchor = AnchorStyles.Top | AnchorStyles.Bottom | AnchorStyles.Left | AnchorStyles.Right;
-            pnlLeft.BackColor = Color.FromArgb(((int)(((byte)(223)))), ((int)(((byte)(228)))), ((int)(((byte)(235)))));
+            pnlLeft.BackColor = Color.FromArgb(((int)(((byte)(223)))), ((int)(((byte)(228)))), ((int)(((byte)(235))))).AdaptBackColor();
```

Comment:
We shouldn't be manually changing the designer files, as these will get overwritten once we open the form in the designer.

Reply:
In this case, the color assign in itself is removed too, so not a difference.
Normally, you need to revert every change by the designer anyway.
I am not sure if something like `Remove.ImageTransparentColor = Color.Magenta.AdaptBackColor();`even need to be changed.
It was easiest to change all occurrences for now, to find the usages.

I would have preferred to split this PR in several sequiential PRs, but there will be feedback back for some time.
There are some blockers (.net in AppVeyor, FileStatusList use TreeView and likely a 5.2 release) before this is ready to merge anyway. But I find this very usable in normal work.